lib/test_suite.py

```diff
@@ -265,5 +265,11 @@ def run_test(self, test, server, inspector):
     def is_parallel(self):
         return self.ini['is_parallel']
 
+    def fragile_retries(self):
+        try:
+            return self.fragile['retries']
+        except Exception:
+            return 0
```

Comment:
Nit: It is the same as `self.fragile.get('retries', 0)`.

Reply:
And catching a general exception is considered as the bad practice.

Reply:
Ok, sure, corrected.